deploy/clowdapp.yml

```
@@ -1265,6 +1264,79 @@ objects:
             "created_on" AS "created_at",
             "modified_on" AS "updated_at"
           FROM "hbi"."groups"
+      - prefix: hms_analytics/inventory/${ENV_NAME}/hosts_metrics/hosts_count
+        query: >-
+          SELECT COUNT(*) AS "hosts_count"
+          FROM "hbi"."hosts";
+      - prefix: hms_analytics/inventory/${ENV_NAME}/hosts_metrics/distinct_accounts
+        query: >-
+          SELECT COUNT(DISTINCT "account") AS "count"
+          FROM "hbi"."hosts";
+      - prefix: hms_analytics/inventory/${ENV_NAME}/hosts_metrics/distinct_org_ids
+        query: >-
+          SELECT COUNT(DISTINCT org_id) AS "count"
```

Does org_id have to be in quotes?
```suggestion
          SELECT COUNT(DISTINCT "org_id") AS "count"
```